src/viewer.js

```diff
@@ -861,6 +861,23 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
     },
 
     /**
+     * Turns debugging mode on or off for this viewer.
+     *
+     * @function
+     * @param {Boolean} true to turn debug on, false to turn debug off.
+     * @return {OpenSeadragon.Viewer} Chainable.
```

Comment:
We're actually moving away from making the calls chainable; let's remove that from this function.